crates/re_renderer/src/resource_managers/texture_manager.rs

```diff
@@ -96,12 +96,29 @@ impl<'a> Texture2DCreationDesc<'a> {
 }
 
 #[derive(thiserror::Error, Debug)]
-pub enum TextureManager2DError<DataCreationError> {
+pub enum TextureCreationError {
```

Comment:
good idea to split it out. we should consider moving this down to the texture pool later :)